src/help/components/entry/Entry.tsx

```diff
@@ -139,7 +147,9 @@ const HelpEntry = ({
   match,
   inPanel,
 }: RouteChildrenProps<{ accession: string }> & Props) => {
-  const accession = match?.params.accession;
+  const [accession, hash] = match?.params.accession?.split(
+    encodeURIComponent('#')
+  ) || [match?.params.accession, undefined];
```

Comment:
Wait, how does the hash end up in the accession? This is supposed to be extracted from the pathname 🤔 
Is this an issue with the way we're generating the links?
It probably needs to split the hash there and pass it as a hash, otherwise the hash sign is probably getting encoded in order to be part of the pathname

Reply:
I played a bit with trying to add it in the path definition `/:accession#:hash?` but didn't get anywhere. Do you know if there's a way?

Reply:
oh hang on, I see what you mean, need to look at `generatePath`

Reply:
Have another look. Still have to do the split but higher up so I can use hash in path

Reply:
I would have done the split in the body of `ContextualHelpContainer` where we need it separated.
To be fair if the reference we put in `articleId` might be an article ID, and article ID + a hash, or even an external URL, I'd rename that state.
But it also depends on where you'd want to put the `window.open` to open a new tab when it's an external URL in the data attribute.